modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/RoleSelectorDropdown/index.jsx

```diff
@@ -0,0 +1,218 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
```

Comment:
Could you go through this file again and check the formatting? There's some extra spaces, missing semicolons, etc.

Reply:
Done.